Backpack-SwiftUI/Card/Classes/BPKCardWrapper.swift

```diff
@@ -36,6 +36,16 @@ public struct BPKCardWrapper<HeaderContent: View, WrappedContent: CardWrappable>
         self.card = card()
     }
     
+    public init(
+        backgroundColor: UIColor,
+        @ViewBuilder header: () -> HeaderContent,
+        @ViewBuilder card: () -> WrappedContent
+    ) {
+        self.backgroundColor = BPKColor.init(value: backgroundColor)
+        self.header = header()
+        self.card = card()
+    }
```

Comment:
Is this change needed in this PR?

Reply:
Definitely not, made it into it somehow when merging stuff 🤔 . Checking 👍